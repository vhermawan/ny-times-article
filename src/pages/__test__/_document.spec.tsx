import { render } from '@testing-library/react';
import Document from '../_document';

jest.mock('next/document', () => ({
  __esModule: true,
  Html: () => <div />,
  Head: () => <div />,
  Main: () => <div />,
  NextScript: () => <div />,
}));

describe('Document component', () => {
  it('renders without error', () => {
    const { container } = render(<Document />);
    expect(container.firstChild).not.toBeNull();
  });
});
