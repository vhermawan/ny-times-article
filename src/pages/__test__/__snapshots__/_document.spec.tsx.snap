// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Document should render correctly 1`] = `
<div>
  <html
    lang="en"
  >
    <body>
      <div
        id="main"
      />
    </body>
  </html>
</div>
`;
